src/modules/tasks.js

```diff
@@ -53,7 +53,8 @@ export default class Tasks {
 
   taskExists = (d) => {
     d = d.toLowerCase().trim();
```

Comment:
According to W3 javascript best practices, the 'd' could either be given a default value [(d='')] OR it is tested inside the function for 'undefined' [if(d !== undefined) {// function code inside}]. This is due to the fact that If a function is called with a missing argument, the value of the missing argument is set to "undefined". Undefined values can break your code. It is a good habit to assign default values to arguments.

Reply:
Hello @mnekx ,
Thanks for the review, yes that's is the best practice, but it implies how the function is called, where the function is called the value of `d` can never be `null` or `undefined`.
Please take a look at where and how the function is called.